sql/electives-form.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+   <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+	<title>Electives Form</title>
+</head>
+<body>
+	<h1>Electives form</h1>
+	<?php include './validation.php';?>
+	<form action="./validation.php" method="POST">
```

Comment:
Why do you include validation.php file in the php snippet and in the action attribute. Maybe you have forgotten to delete the php snippet.